src/actions/getCurrentVersionsByPackage.js

```diff
@@ -0,0 +1,10 @@
+export function getCurrentVersionsByPackage({ packageJson, props }) {
```

Comment:
And we have to use the keys in `packagesPaths` to get all versions, not just the ones that change so that we can update `package.json` dependencies properly.